_providers/old/aol.md

```diff
@@ -1,19 +1,20 @@
 ---
 name: Aol. (America Online)
+domains: 
+  - aol.com
+status: BROKEN
+servers:
+  - type: imap
+    socket: SSL
+    hostname: export.imap.aol.com
+    port: 993
+  - type: smtp
+    socket: SSL
+    hostname: smtp.aol.com
+    port: 465
+before_login_hint: |
+  You have to send at least one email from the web interface before logging in works.
```

Comment:
The text reads more like an explanation for `PREPARATION`. Can we explain shortly why it's `BROKEN`?

Reply:
I left the file in the old/ folder, so it doesn't *really* matter for this PR. It was basically the state of my findings when I realised that AOL uses OAuth now, which needs work in core, afaik.

And depending on #91 it might be a PREPARATION provider soon. I'm not really sure about this before_login_hint anyway...

Reply:
I pushed another commit which makes clear that these are only notes for when we really get it working.